CHANGELOG.md

```diff
@@ -2,6 +2,15 @@
 
 All notable changes to this project will be documented in this file.
 
+## [1.22.3](https://github.com/voxpupuli/beaker-puppet/tree/1.22.3) (2022-02-22)
+
+[Full Changelog](https://github.com/voxpupuli/beaker-puppet/compare/1.22.2...1.22.3)
+
+**Merged pull requests:**
+
+- \(maint\) Update specs to fix ruby 3 spec failures [\#174](https://github.com/voxpupuli/beaker-puppet/pull/174) ([cthorn42](https://github.com/cthorn42))
+- \(maint\) Update agent download from URL logic to handle redirects [\#173](https://github.com/voxpupuli/beaker-puppet/pull/173) ([cthorn42](https://github.com/cthorn42))
```

Comment:
Should we rename the PR to remove `(maint) ` from the title?  Not sure it is valuable for end-users.

I feel like #173 is an enhancement, if so it should be labeled as so and the minor version should be bumped according to semver.

Reply:
Okay I updated the change log and need to fix the minor version bump now.

Reply:
Thanks!